### ISSUE TITLE
Windows: 8192 character command line length limit

### ISSUE THREAD

**Issue by G100g** [ahmednuaman/grunt-scss-lint#127]
Hi,
 on windows, passing each file to command line, cause droping some files path when you have a lot of file to lint.

So, linter report green light, but some file are missing from linting.

We have, for example, 198 files path and the last fifty files are not linted.

Could be a solution, copy conf file, write a temporary one and insert all file paths inside "scss_files" option in the file?

Thank you

**Comment by G100g** [comment]
Actually my solution is creating different task configurations, splitting files across multiple scss_lint command line calls.

Work also using newer plugin

**Comment by ahmednuaman** [comment]
PR?

**Comment by G100g** [comment]
After tried various ideas, I preferred to don't modify grunt-scss-lint files.
I changed configuration in my grunt file, splitting in different tasks

```javascript
scsslint: {
  desktop: [ 'desktop/**/*scss' ],
  mobile: [ 'mobile/**/scss' ],
  commons: [ 'commons/**/*scss' ]
}
```

With this configuration I can use newer plugin on my watch task, to run scsslint only on updated files.

The idee to use a temp  yml file, is not good for using newer plugin, and change lint function to run multiple instance, I think only add complexity to the code.

But maybe i'm wrong :)
